src/uct/ib/dc/verbs/dc_verbs.c

```diff
@@ -1246,6 +1247,16 @@ static UCS_CLASS_INIT_FUNC(uct_dc_verbs_iface_t, uct_md_h md, uct_worker_h worke
     }
     uct_dc_iface_set_quota(&self->super, &config->super);
 
+    rx_qlen_init = config->super.ud_common.rx_queue_len_init;
+    ucs_mpool_grow(&self->super.super.rx.mp,
```

Comment:
maybe this needs to be moved to dc_iface?

Reply:
not yet, because dc_mlx5 is not using this